### DISCUSSION TITLE
Balance not Updating when Purchasing a Plan

### DISCUSSION THREAD

**Discussion by joesixpack** [subquery/subql#952] [Frontier TestNet - Bug Reporting]
![image](https://user-images.githubusercontent.com/23145642/165996083-bc66de1e-9922-46c2-9d99-d6568c84abcd.png)

![image](https://user-images.githubusercontent.com/23145642/165996147-8caa9ae1-3297-4d3b-84f4-5f7b8592a9eb.png)

**Comment by joesixpack** [comment]
Ah, seems like its not clear that a "price for a plan" is actually a total cost and not a per token cost.  Phrasing should be changed.

**Reply by heyacherry** [reply]
Hello @joesixpack 
Will have a think on wordings, thanks!

**Comment by lilian79511** [comment]
Ticket is created: SQN-418
